Automattic/jetpack
# Blaze: use translated post type name in post-publish panel

## Proposed changes:

Instead of relying on the raw post type slug, let's pull the translated post type label. We need to ensure it is lowercase, since it's used in a sentence.

| **Before** | **After** |
|  ---  |  ---  |
|          <img width="299" alt="Screenshot 2023-06-15 at 14 37 03" src="https://github.com/Automattic/jetpack/assets/426388/a7e77a8a-6415-471a-9d38-e9919875ead4">          |        <img width="285" alt="Screenshot 2023-06-15 at 14 40 42" src="https://github.com/Automattic/jetpack/assets/426388/f058ce1f-09b8-4fcf-a9f4-449ce8bf104e">         |


> **Note**
> In the French translation above, I had to move the non-translated post type name at the end of the string, where it looked less worse standing on its own. Once this fix is merged and shipped, I will be able to update the translated string again to form a proper sentence without parenthesis.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Go to Settings > General, and switch your site language to French.
* Go to Dashboard > Updates, and update translations if necessary.
* Go to Posts > Add New, write a new post.
* Publish your post.
* View the post-publish panel and its Blaze section; the post type name should be translated.
